Code/Tools/ProjectManager/Platform/Mac/ProjectExportWorker_mac.cpp

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <ProjectExportWorker.h>
+#include <ProjectManagerDefs.h>
+#include <ProjectUtils.h>
+
+#include <QDir>
+#include <QString>
+
+namespace O3DE::ProjectManager
+{
+    QString ProjectExportWorker::GetO3DECLIString() const
+    {
+        return QString("scripts/o3de.sh");
```

Comment:
Crazy, how does project manager gui call o3de.sh/bat without a method like this already existing?

Reply:
It's possible it's using `o3de` as the shorthand, which works on windows, but I've run into issues on linux where if there is a twin directory the terminal gets confused, and you have to add .sh anyways.

Reply:
On windows, there is an environment variable called `PATHEXT` that it uses to run a shorthanded command. On my windows system, it looks like:
```
PATHEXT=.COM;.EXE;.BAT;.CMD;.VBS;.VBE;.JS;.JSE;.WSF;.WSH;.MSC;.PY;.PYW
```
That means that when I type in `o3de` at the command line, it will search for the `o3de` and use the first one that it finds in is `%PATH%` environment variable that has the first extension in that `%PATHEXT%` setting. On linux, you have to be explicit and you must have the `x` (executable) permission flag set.